helpers/Matrix.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _HELPERS_MATRIX_H_
+#define _HELPERS_MATRIX_H_
+
+#include "../include/z64.h"
+#include "../symbols.h"
```

Comment:
```suggestion
#include "include/z64.h"
#include "symbols.h"
```
The include path should include the z64hdr repo, right?